src/lib/kit/components/Inputs/Checkbox/__tests__/Checkbox.visual.test.tsx

```diff
@@ -30,6 +30,12 @@ test.describe('Checkbox', () => {
 
         await expectScreenshot();
     });
+
+    test('layout transparent', async ({mount, expectScreenshot}) => {
```

Comment:
you need to take a screenshot, to do this you need to run the command `npm run playwright:docker`